content/en/blog/2023/otel-end-user-discussions-january-2023/index.md

```diff
@@ -0,0 +1,245 @@
+---
+title: OpenTelemetry End User Discussions Summary for January 2023
+linkTitle: End User Discussions Jan 2023
+date: 2023-01-25
+author: >-
+  [Adriana Villela](https://github.com/avillela) (Lightstep)
+---
+
+Each month, users in the OpenTelemetry (OTel) community gather to talk about how
+they use OpenTelemetry in real life. Sessions are held for users in the Americas
+(AMER), Europe Middle-East & Africa (EMEA), and Asia-Pacific (APAC). The
+discussions take place using a
+[Lean Coffee format](https://agilecoffee.com/leancoffee/), whereby folks are
+invited to post their topics to the
+[Agile Coffee board like this one](http://agile.coffee/#b3b37364-d40e-4029-847c-8ee059d60855),
+and everyone in attendance votes on what they want to talk about.
+
+This is a great way to meet other users in the OpenTelemetry community, and to
+learn about and share practical experience on how OpenTelemetry is being used in
+the wild. Each meeting is attended by an OTel Governance Committee member and/or
+maintainer to help answer questions, listen to user feedback, and provide
+additional context and insight into the topics discussed.
+
+This is the first in a series of blog posts summarizing our monthly OTel End
+User Discussions, starting with our January 2023 sessions.
+
+### What we talked about
```

Comment:
Can you **promote _all_ headings** up a level, so that the page's top-level headings are at level 2 (`##`) -- like this:

```suggestion
## What we talked about
```

This will give the page a more meaningful table of contents.

Reply:
Done!